database/schema.sql

```diff
@@ -141,4 +141,5 @@ GRANT INSERT ON analysis, certificates, scans, trust TO tlsobsscanner;
 GRANT UPDATE ON analysis, certificates, scans, trust TO tlsobsscanner;
 GRANT USAGE ON analysis_id_seq, certificates_id_seq, scans_id_seq, trust_id_seq TO tlsobsscanner;
 
-ALTER TABLE certificates ADD COLUMN x509_extendedKeyUsageOID jsonb NULL;
\ No newline at end of file
+ALTER TABLE certificates ADD COLUMN x509_extendedKeyUsageOID jsonb NULL;
+ALTER TABLE certificates ADD COLUMN mozillaPolicyV2_5 jsonb NULL;
```

Comment:
could you add this new column to the create table certificate block, and move the alter table to `database/migrations/1.3.3.sql` ?